### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#142] [Esercizi - Bertini] [answered]
![Es 8](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/60dded7f-de05-423d-8a0c-bd2fc4b5c10b)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/46ae109d-8347-4e1c-972f-9f70548804b3)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione discussa in classe
- $\Omega = \\{(w_0,w_1)| 1 \leq w_i \leq 6\\}, w_0 = rosso, w_1= blu$
- $A = \\{(w_0,w_1) | w_0=5\\}$
$B= \\{(w_0,w_1)| w_0=5 \lor w_1=5\\}$
$C =\\{(w_0,w_1)| w_0=w_1=5\\}$
$D = \\{(w_0,w_1)| w_0 \neq 5 \land w_1 \neq 5\\} = B^c$ 
$E = \\{(w_0,w_1)| w_0 +w_1 = 5\\}$
- $|\Omega| = 6\cdot 6=36$
$\mathbb{P}(A)= \frac{|A|}{|\Omega|}=6/36=1/6$
$\mathbb{P}(B)= 1-\frac{|B^c|}{|\Omega|}=1-(25/36)=11/36 $
$\mathbb{P}(C)= \frac{|C|}{|\Omega|}=1/36$
$\mathbb{P}(D)= \frac{|D|}{|\Omega|}=25/36$
$\mathbb{P}(E)= \frac{|E|}{|\Omega|}= 4/36 = 1/9$
